dashboard/config/locales/scripts.en.yml

```diff
@@ -7,6 +7,7 @@ en:
         csd_category_name: CS Discoveries ('17-'18)
         csd_2018_category_name: CS Discoveries ('18-'19)
         csf_category_name: CS Fundamentals
+        csf_2018_category_name: CS Fundamentals (2018)
```

Comment:
Should this be "('18-'19)" as well?

Reply:
Great question! These are different because CSP and CSD are aligned with the school year and CSF courses are not. Ryan agreed with "2017" and "2018" when I proposed it to him, but I'm not sure he was aware of this alternative. I'll double-check with him before merging.

Reply:
Makes sense!